### DISCUSSION TITLE
How to freeze grid's header and some of the columns at the left?

### DISCUSSION THREAD

**Discussion by DevotedCode** [nick-keller/react-datasheet-grid#104] [Q&A] [answered]
Is there a way to customize the grid that its header and specified columns at the left side will be always visible even when user scrolls grid's content?

**Comment by nick-keller** [comment, marked as answer]
Hello @DevotedCode,
The header row is always visible / fixed / sticky, so is the left gutter column. You can also add a sticky column to the right.
But you cannot add more sticky rows or columns, this use case is too complex for the current architecture, notably making the selection work across sticky and non-sticky content is complicated.
